modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/utils/string.ts

```diff
@@ -25,6 +25,32 @@ export function endStringWithForwardSlash(str?: string) {
 	return str;
 }
 
+/*
+ * Takes all the path without the last parameter
+ */
+export function getAllButLastParameterFromPath(path: string) {
+	if (path) {
+		return path.lastIndexOf('/{') > 0
+			? path.slice(0, path.lastIndexOf('/{'))
+			: path;
+	}
+
+	return '';
+}
+
+/*
+ * Takes the last parameter of a path
+ */
+export function getLastParameterFromPath(path: string) {
+	if (path) {
```

Comment:
Try if this works, otherwise this is ok: 

`return path?.includes('/') ? path.substring(path.lastIndexOf('/') : '';`